### PR TITLE
Make Battery notification timeout configurable

### LINKED ISSUE #3353: Battery low notifications go away with `dunst`
yesterday, thanks to @m-col, I manage to have beautiful popup notifications for my battery from within `qtile`'s bar and even sound and brightness bars as I change them with the keyboard:
notifications dunst
However it is still not behaving quite as I expected.

I expected the notifications for a low battery to be of *urgent*/*critical* priority and it is red indeed.
I configured my [dunst](https://github.com/a2n-s/dotfiles/blob/main/.config/dunst/dunstrc#L292) so that *low* and *normal* notifications go away quite quicly, but *critical*/red ones never go away!

This is what I observe when I run either
```
notify-send "ERROR" "this is an error" -u critical
notify-send "NORMAL" "this is a normal message" -u normal
notify-send "LOW" "this is nothing important" -u low
```
or
```
dunstify "ERROR" "this is an error" -u critical
dunstify "NORMAL" "this is a normal message" -u normal
dunstify "LOW" "this is nothing important" -u low
```

But this is **not the case with the *critical* battery low notifications** from `qtile`.

I do not really understand the code of the `libqtile.widget.Battery` class, could anyone help fix that?

### PR DESCRIPTION
**Pull request by elParaguayo**
Users may want a different timeout other than the default 10 seconds.

Closes #3353

### CODE REVIEW

**Comment by m-col** [comment]
Changes itself LGTM. The test failure is unrelated but cancelled the py310 tests so I've run them again.

File: libqtile/widget/battery.py [hunk 1]
```diff
@@ -345,6 +346,7 @@ def __init__(self, **config) -> None:
 
         self._battery = self._load_battery(**config)
         self._has_notified = False
+        self.timeout = int(self.notification_timeout * 1000)
```
Comment (m-col) [inline libqtile/widget/battery.py:349]: Maybe we can do away with this line and accept a millisecond argument

Comment (m-col) [reply libqtile/widget/battery.py:349]: Oh nevermind, I merged :laughing:

Comment (elParaguayo) [reply libqtile/widget/battery.py:349]: I thought about that. Kept it as seconds as that's consistent with `update_interval` for other widgets.

**Pull request merged**